catalog/region-restriction-templates/RestrictToSpecifiedRegions.yaml

```
@@ -28,6 +28,9 @@
     - "iam:*"
     - "importexport:*"
     - "kms:*"
+    %{ if cloudfront_lambda_edge_enabled == "true" }
+    - "lambda:*"
+    %{ endif }
```

Same idea as with `DenyLambdaInRegions`

```suggestion
%{~ if cloudfront_lambda_edge_enabled && !strcontains(allowed_regions, "us-east-1") ~}
	    - "lambda:*"
    %{~ endif ~}
```